.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,49 @@
-Fixes #
-
-### Changes proposed in this Pull Request
+Resolves #
 
+## Proposed Changes
 *
 
-### Testing instructions
-
+## Testing Instructions
 <!--
-If the functionality of this PR is behind a feature flag, please include the
-following testing step, using the correct name for the feature flag. (If you
-aren't sure, just ignore this step)
-
-* Enable the feature flag: `add_filter( 'sensei_feature_flag_{THE_FLAG_NAME}', '__return_true' );`
+Add as many details as possible to help others reproduce the issue and test the fix.
```

Comment:
As we've changed it from **Fixes** to **Resolved** at the top, should we also change it here?

Reply:
https://github.com/Automattic/sensei/pull/6542/commits/c6aaf1e670e8e3d9b7ada45a1352a2fd2b790400